.github/workflows/lint-format.yml

```diff
@@ -0,0 +1,54 @@
+name: Lint & Format
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**.m?js'
+      - '**.ts'
+      - '**.json'
+      - '!package-lock.json'
+      - '**.ya?ml'
+      - '**.md'
+      - '.editorconfig'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.m?js'
+      - '**.ts'
+      - '**.json'
+      - '!package-lock.json'
+      - '**.ya?ml'
+      - '**.md'
+      - '.editorconfig'
```

Comment:
what does this really exclude? 😄

Reply:
It was more of an attempt 😄😄 Probably going to revert...